### PR TITLE
feat: add new nav dropdown for admin view

### PR DESCRIPTION
**Pull request by carissa-tang**
## Implementation description. How did you make this change?
* Add dropdown UI in navigation header w/ custom style
* Change `/admin/view-donations` to`/view-donations`
* Create a new route called `/admin/check-ins`

## Steps to test
1. Sign in using admin account (`name@uwblueprint.org` email)
2. Test Navigation bar
3. Check Figma designs

https://user-images.githubusercontent.com/69697180/158032753-c4aa8185-c1ec-4c8e-a340-b05cedabc667.mov
image

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s) 
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [x] The appropriate tests if necessary have been written

### CODE REVIEW

File: frontend/src/components/common/Header.tsx [hunk 1]
```diff
@@ -100,9 +118,52 @@ const Header = (): JSX.Element => {
...
+                        _focus={{
+                          bg: "raddish.50",
+                        }}
+                        _active={{
+                          bg: "raddish.50",
+                        }}
+                      >
+                        <Link as={ReactLink} to={Routes.ADMIN_CHECK_INS}>
+                          Fridge Check-ins
+                        </Link>
```
Comment (hanlinc27) [inline frontend/src/components/common/Header.tsx:146]: small detail and you might want to check with @stacykwok but the whole menu should be clickable rather than just the linked text right?
You can probably just remove the `Link` element and wrap the menu item with a clickable div or something else to route 👍🏻 
https://user-images.githubusercontent.com/19617248/158285011-b8ce7df1-bd81-45e2-86c0-760b6701973c.mov

File: frontend/src/components/pages/ViewDonations.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const ViewDonations = ({
   };
 
   return (
-    <Container alignContent="left" variant="calendarContainer">
+    <Container alignContent="left" variant="responsiveContainer">
```
Comment (hanlinc27) [inline frontend/src/components/pages/ViewDonations.tsx:50]: can we revert this change? Calendar isn't aligned otherwise: 
image

**Review by Shehryar21 — COMMENTED** [review]
two quick points:
1. Admin routes should be restricted to admin only, donors and volunteers can access right now
2. Can you add redirecting to home page when logging out in this pr, should be quick?

**Review by hanlinc27 — APPROVED** [review]
